October updates

thetwopct opened the pull request:
1. Tidying up files and comments ready for handover
2. Fix PHP Notice:  Undefined property: stdClass::$twitter in sync-projects.php
3. #604 Banner Slider refactor
4. Adding ACF Pro Plugin
5. #590 Creating Gallery Slider ACF Block and associated setup

I've installed ACF as a pre-cursor for the new dev team, and as part of the Tier 3 Report improvements.

I've added ACF Pro to the repo (it doesn't work well with Composer from what I researched).

I've used my copy of [ACF Pro](https://www.advancedcustomfields.com/pro/#pricing-table), it will need a licence. I believe a basic licence includes one website (development/staging and production) which should cover us.

cjyabraham commented:
- I think it should be "Upcoming Events" and not "Forthcoming Events"... not sure if that's an English-ism but "forthcoming" is kind of formal, as in "Your lady is forthcoming," 2 HENRY VI., ii. 1. 174
- Also I think "THE NEXT CNCF EVENT" should be "Upcoming Event" as it may not necessarily be the next event CNCF is hosting but just the next one we want to push
- I do get a console error when there is more than one event, however, the slider still works
- There is a HR above and below the event slider when there is more than one event. When there is only one event you don't have an HR below the banner. This should probably be consistent? I'm not sure the HR below the banner/slider is needed so my vote would be to remove it in both cases but perhaps we should just stick with the design?

thetwopct commented:
1. 2. OK. Englishism vs Americanism I guess. Updated.
3. Fixed I think, stupidly simple fix when coming back to this with fresh eyes.
4. Made consistent.
